.github/workflows/new-issue.yml

```diff
@@ -9,21 +9,106 @@ name: "new-issue"
+        run: echo "::set-output name=STEPS_REPRODUCE::$STEPS_REPRODUCE"
+      - name: Missed Link to a GitHub repository
+        id: link_to_github
+        if: contains(github.event.issue.body, 'Repository to reproduce') == false
+        env:
+          LINK_TO_GITHUB: Link to a GitHub repository
+        run: echo "::set-output name=LINK_TO_GITHUB::$LINK_TO_GITHUB"
+      - name: Missed Expected result
+        id: expected_result
+        if: contains(github.event.issue.body, 'Expected result') == false
+        env:
+          EXPECTED_RESULT: Expected result
+        run: echo "::set-output name=EXPECTED_RESULT::$EXPECTED_RESULT"
+      - name: Missed Actual result
+        id: actual_result
+        if: contains(github.event.issue.body, 'Actual result') == false
+        env:
+          ACTUAL_RESULT: Actual result
+        run: echo "::set-output name=ACTUAL_RESULT::$ACTUAL_RESULT"
+      - name: Missed CLI version
+        id: missed_cli_version
+        if: contains(github.event.issue.body, 'CLI Version') == false
+        env:
+          CLI_VERSION: CLI version
+        run: echo "::set-output name=CLI_VERSION::$CLI_VERSION"
+      - name: Missed plugin version
+        id: missed_plugin_version
+        if: contains(github.event.issue.body, 'plugin Version') == false
+        env:
+          PLUGIN_VERSION: Plugin version
+        run: echo "::set-output name=PLUGIN_VERSION::$PLUGIN_VERSION"
+      - name: Missed OS version
+        id: missed_os_version
+        if: contains(github.event.issue.body, 'OS') == false
+        env:
+          OS_VERSION: OS version
+        run: echo "::set-output name=OS_VERSION::$OS_VERSION"
+  display-message:
+    runs-on: ubuntu-latest
+    needs: [triage, missed-info]
+    steps:
+      - uses: actions/checkout@v2.0.0 
+      - name: issue comment
+        if: ${{needs.missed-info.outputs.STEPS_REPRODUCE != '' 
+          || needs.missed-info.outputs.LINK_TO_GITHUB != ''
+          || needs.missed-info.outputs.EXPECTED_RESULT != ''
+          || needs.missed-info.outputs.ACTUAL_RESULT != ''
+          || needs.missed-info.outputs.CLI_VERSION != ''
+          || needs.missed-info.outputs.PLUGIN_VERSION != ''
+          || needs.missed-info.outputs.OS_VERSION != ''}}
+        id: issue_comment_missing
+        uses: ./
+        with: 
+          repo-token: ${{ secrets.GITHUB_TOKEN}}
+          message: >
+            ${{format('Your issue is missing this required information: {0} {1} {2} {3} {4} {5} {6}. When you provide us this information we''ll triage and reproduce the issue.',
+            needs.missed-info.outputs.STEPS_REPRODUCE, needs.missed-info.outputs.LINK_TO_GITHUB, needs.missed-info.outputs.EXPECTED_RESULT, needs.missed-info.outputs.ACTUAL_RESULT,
+            needs.missed-info.outputs.CLI_VERSION, needs.missed-info.outputs.PLUGIN_VERSION, needs.missed-info.outputs.OS_VERSION)}}
+      - name: issue comment
+        if: ${{needs.missed-info.outputs.STEPS_REPRODUCE == '' 
```

Comment:
I think we want to add this message regardless if the issue is missing peices.

Reply:
Ok, i can take out that if then.